commands/test/test.go

```diff
@@ -187,27 +200,32 @@ func runRules(ctx context.Context, namespace string, input interface{}, regex *r
 			return nil, err
 		}
 
-		totalErrors = append(totalErrors, errors...)
+		if regex != nil {
+			totalErrors = append(totalErrors, errors...)
+		} else {
+			errors := []error{fmt.Errorf("rule %s", rule)}
+			totalErrors = append(totalErrors, errors...)
+		}
 	}
 
 	return totalErrors, nil
 }
 
 func getRules(ctx context.Context, re *regexp.Regexp, compiler *ast.Compiler) []string {
-	var res []string
+	var rules []string
```

Comment:
Thanks for this! I think this makes it a bit more clear on what we're storing.

Reply:
Take a look at this pr for this, please:)
https://github.com/instrumenta/conftest/pull/162

Reply:
merged!